urbanairship-react-native/ios/UARCTModule/UrbanAirshipReactModule.m

```diff
@@ -351,13 +351,13 @@ - (instancetype)init{
 
 RCT_EXPORT_METHOD(displayMessageCenter) {
     dispatch_async(dispatch_get_main_queue(), ^{
-        [[UAMessageCenter shared] display];
```

Comment:
This should still be display, what we need to fix is the delegate that we set needs to check if autoLaunchMessageCenter is enabled, and if so show the default UI